docs/source/sycamore.md

```diff
@@ -189,29 +186,3 @@ agg_data = aggregate_surveys_config(study_dir, config_path, study_tz, history_pa
 ca_detail, ca_summary = agg_changed_answers_summary(config_path, agg_data)
  
 ```
-
-#FAQ:  
-
-#### In the `submits_summary.csv` file, there are some rows where `num_submitted_surveys` is greater than `num_surveys`. How could a user have submitted more surveys than were delivered to them?  
```

Comment:
The documentation build error was:
```
/home/docs/checkouts/readthedocs.org/user_builds/forest-docs/checkouts/latest/docs/source/sycamore.md:195: WARNING: Non-consecutive header level increase; H2 to H4 [myst.header]
/home/docs/checkouts/readthedocs.org/user_builds/forest-docs/checkouts/latest/docs/source/sycamore.md:201: WARNING: Non-consecutive header level increase; H2 to H4 [myst.header]
/home/docs/checkouts/readthedocs.org/user_builds/forest-docs/checkouts/latest/docs/source/sycamore.md:205: WARNING: Non-consecutive header level increase; H2 to H4 [myst.header]
```
https://readthedocs.org/projects/forest-docs/builds/19975506/

This is due to jumping from a level 2 heading (`##`) to a level 4 (`####`) here (https://www.markdownguide.org/basic-syntax/#headings)
Note that the level 1 heading `#FAQ` was ignored because of a missing space after `#` (https://www.markdownguide.org/basic-syntax/#heading-best-practices)

You can fix this by using something like an unordered list (https://www.markdownguide.org/basic-syntax/#unordered-lists) or a definition list (https://www.markdownguide.org/extended-syntax/#definition-lists).